#! /bin/sh /usr/share/dpatch/dpatch-run
## 10fstab.dpatch by  <lamont@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: create /etc/mtab with mode 0600

@DPATCH@
diff -urNad util-linux/mount/fstab.c /tmp/dpep.pHan4D/util-linux/mount/fstab.c
--- util-linux/mount/fstab.c	2004-12-04 16:20:19.000000000 -0700
+++ /tmp/dpep.pHan4D/util-linux/mount/fstab.c	2004-12-15 07:42:42.193006107 -0700
@@ -462,7 +462,7 @@
 		struct flock flock;
 		int errsv, fd, i, j;
 
-		i = open (linktargetfile, O_WRONLY|O_CREAT, 0);
+		i = open (linktargetfile, O_WRONLY|O_CREAT, 0600);
 		if (i < 0) {
 			int errsv = errno;
 			/* linktargetfile does not exist (as a file)
